workflow_creation/utilities_plm_webclient.md

```diff
@@ -1,41 +1,228 @@
+
+<p align="center">
+  <img src="img_plm/upload_file_form.png"
+   alt="Form to upload a file" />
+</p>
+
+A file is selected by clicking the "Browse..." button. Select the file type from the "Type" drop-down list. The description of the file is mandatory.
+Click "OK" to start upload. After a while the uploaded file will appear in the "Files" tab of the selected folder.
+
+### Export ZIP package
+
+The content of a folder and all children below including their documents can be exported as a ZIP package.
+Click <img src="img_plm/open_icon.png" /> icon of a node and select the "Download" menu item from the context menu.
+
+### Add remark
+
+A folder can be commented, that is, a new remark may be added.
+Click <img src="img_plm/open_icon.png" /> icon of a folder and select the "Add remark" menu item from the context menu.
+
+<p align="center">
+  <img src="img_plm/add_remark.png"
+   alt="Form to add a remark" />
+</p>
+
+Give the subject of your remark and the remark itself. Then click "OK".
+
+### Add approval
+
+A folder and all its underlying content can be approved (disapproved) by adding an approval tag to the folder.
+Click <img src="img_plm/open_icon.png" /> icon of a folder and select the "Add approval" menu item from the context menu.
+
+<p align="center">
+  <img src="img_plm/add_approval.png"
+   alt="Form to add an approval" />
+</p>
+
+Select the resolution type (approved, rejected or not applicable) and add a description.
+Then click "OK".
+
+### Import folder (ZIP package)
+
+A local folder and all its content (including sub folders) can be imported to a selected node. This, however, requires that the local folder has been zipped first.
+Click <img src="img_plm/open_icon.png" /> icon of a node and select the "Import folder" menu item from the context menu.
+
+<p align="center">
+  <img src="img_plm/import_folder.png"
+   alt="Form to import folder" />
+</p>
+
+Select "Folders type" item. All imported folders will be assigned this type.
+"Files types" may be empty. In this case the system will set the file type (for each file) according to the individual file extension.
+
+### Delete folder
+
+A folder can be deleted. That is, it can be removed from the product structure.
+Click <img src="img_plm/open_icon.png" /> icon of a node and select the "Delete" menu item from the context menu.
+
+<p align="center">
+  <img src="img_plm/delete_confirmation.png"
+   alt="Delete confirmation in Mozilla Firefox" />
+</p>
+
+# Integrating the PLM web client in a workflow
+The displayHtml service is currently registered under the following URI:
+```
+http://www.caxman.eu/apps/jotne/IframeDisplay.owl#displayHtml_Service
+```
+Use this URI to add the service to your workflow.
+
+The following screen shot shows the minimal connections you need to make to
+the PLM file chooser service block:
+<p align="center">
+  <img src="img_plm/wf_editor.png"
+   alt="Minimal connections made to the PLM file chooser service" />
+</p>

```

Comment:
I cannot comment on the picture itself since it's not part of this pull request, but the workflow input "sessionToken" is wired to "sessionToken" _and_ "serviceID", which is wrong.

Reply:
For some reason I cannot have rights to update the "PLM_File_Chooser" workflow